Description: Avoid some incompatible with catch framework
 Remove an include directive for reporter from the last version of catch
Author: Nicholas Guriev <guriev-ns@ya.ru>
Last-Update: 2017-12-02

--- a/Telegram/SourceFiles/base/tests_main.cpp
+++ b/Telegram/SourceFiles/base/tests_main.cpp
@@ -20,7 +20,6 @@
 */
 #define CATCH_CONFIG_RUNNER
 #include "catch.hpp"
-#include "reporters/catch_reporter_compact.hpp"
 #include <QFile>
 
 namespace base {
--- a/Telegram/gyp/Telegram.gyp
+++ b/Telegram/gyp/Telegram.gyp
@@ -82,7 +82,6 @@
       'codegen.gyp:codegen_lang',
       'codegen.gyp:codegen_numbers',
       'codegen.gyp:codegen_style',
-      'tests/tests.gyp:tests',
     ],
 
     'defines': [
@@ -142,5 +141,14 @@
         ],
       }],
     ],
+  },{
+    # TODO: Make this target more respectful of the `notest` build profile.
+    # There should be no tests when the default target is executed.
+    'target_name': 'check',
+    'type': 'none',
+    'dependencies': [
+      'Telegram',
+      'tests/tests.gyp:tests',
+    ],
   }],
 }
